regolith/tools.py

```diff
@@ -448,3 +451,27 @@ def number_suffix(number):
     else:
         suffix = {1: "st", 2: "nd", 3: "rd"}.get(number % 10, "th")
     return suffix
+
+
+def dereference_institution(input_record, gtx):
+    """Tool for dereferencing institutions"""
```

Comment:
the docstring is a bit terse.

I am not sure what arguments to use...I guess ``input_record`` is some ``presentations``  or ``papers`` collection or subcollection? Or what?  Nothing is returned, but ``input_record`` is updated in place?

"dereference" is a bit jargony.  OK in the function name, but what does it mean in the docstring?